src/SFA.DAS.EmployerIncentives.Api/Controllers/EmploymentCheckController.cs

```diff
@@ -29,6 +32,15 @@ public async Task<IActionResult> Update([FromBody] UpdateEmploymentCheckRequest
             return Ok($"/employmentchecks/{updateRequest.CorrelationId}");
         }
 
+        [HttpPut("/employmentchecks/refresh")]
```

Comment:
This is not a good REST endpoint.  Please use the existing Jobs endpoint and create a new RefreshEmploymentChecks job since that is what is required by the caller.